specs/regression.txt

```diff
@@ -525,6 +525,64 @@ ISSUE 351
 <p>&quot;)</p>
 ````````````````````````````````
 
+ISSUE 285
```

Comment:
Seems as a bad issue ID to me. (Ditto for the other tests below in this patch.)

Reply:
Great catch!